scss/bitstyles/ui/section-heading.stories.mdx

```diff
@@ -1,6 +1,7 @@
+import SectionHeading from './SectionHeading';
 import { Canvas, Meta, Story } from '@storybook/addon-docs';
 
-<Meta title="UI/Content/Section heading" />
+<Meta title="UI/Content/Section heading" component={SectionHeading} />
```

Comment:
This needs to have a unique `title` attribute — it’s the same as the `.js` file, so storybook doesn’t know which is which and renders neither of them in the navigation. The convention we’ve followed elsewhere is that this mdx file is the “overview”

```suggestion
<Meta title="UI/Content/Section heading/Overview" component={SectionHeading} />
```